src/common/com.smi.microservices.common/src/main/java/org/smi/common/rabbitMq/RabbitMqAdapter.java

```diff
@@ -76,32 +66,23 @@ public RabbitMqAdapter(RabbitOptions options, String microserviceName) throws IO
 		_hostId = _headerFactory.getProcessName() + _headerFactory.getProcessId();
 	}
 
-	private void CheckValidServerSettings(ConnectionFactory _factory) throws IOException, TimeoutException {
-		try {
-			if (!_conn.getServerProperties().containsKey("version"))
-				throw new IOException("Could not get RabbitMQ server version");
-
-			String version = ((LongString) _conn.getServerProperties().get("version")).toString();
-			String[] split = version.split(Pattern.quote("."));
-
-			if (Integer.parseInt(split[0]) < MinRabbitServerVersionMajor
-					|| Integer.parseInt(split[1]) < MinRabbitServerVersionMinor
-					|| Integer.parseInt(split[2]) < MinRabbitServerVersionPatch)
-				throw new IOException("Connected to RabbitMQ server version " + version + " but minimum required is "
-						+ MinRabbitServerVersionMajor + "." + MinRabbitServerVersionMinor + "."
-						+ MinRabbitServerVersionPatch);
-
-			_rabbitMqServerVersion = version;
-		} catch (ConnectException e) {
-			StringBuilder sb = new StringBuilder();
-			sb.append("    HostName:                       " + _factory.getHost() + System.lineSeparator());
-			sb.append("    Port:                           " + _factory.getPort() + System.lineSeparator());
-			sb.append("    UserName:                       " + _factory.getUsername() + System.lineSeparator());
-			sb.append("    VirtualHost:                    " + _factory.getVirtualHost() + System.lineSeparator());
-			sb.append("    HandshakeContinuationTimeout:   " + _factory.getHandshakeTimeout() + System.lineSeparator());
-			String excString = String.format("Could not create a connection to RabbitMQ on startup:%n" + sb);
-			throw new RuntimeException(excString, e);
-		}
+	private void CheckValidServerSettings() throws IOException {
+		final int MinRabbitServerVersionMajor = 3;
+		final int MinRabbitServerVersionMinor = 7;
+		final int MinRabbitServerVersionPatch = 0;
+
+		if (!_conn.getServerProperties().containsKey("version"))
+			throw new IOException("Could not get RabbitMQ server version");
+
+		_rabbitMqServerVersion = _conn.getServerProperties().get("version").toString();
+		String[] split = _rabbitMqServerVersion.split(Pattern.quote("."));
+
+		if (Integer.parseInt(split[0]) < MinRabbitServerVersionMajor
```

Comment:
There's similar logic in the c# code which we should also fix if it's broken.

Reply:
Good point - fixed in this PR now.